### PR TITLE
use y2users for local users and groups

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/include/users/complex.rb [hunk 1]
```diff
@@ -26,6 +26,13 @@
 #		Jiri Suchomel <jsuchome@suse.cz>
 #
 # $Id$
+
```
Comment (joseivanlopez) [inline src/include/users/complex.rb:29]: I would not include all these changes related to the integration of Y2Users and Yast::Users. I see this is useful for testing, but there is a separate PR for such an integration.

File: src/modules/Users.pm [hunk 2]
```diff
@@ -3986,6 +3996,7 @@ sub WriteCustomSets {
 ##------------------------------------
 # Writes settings to /etc/defaults/useradd
 sub WriteLoginDefaults {
+    return 1;
```
Comment (joseivanlopez) [inline src/modules/Users.pm:3999]: The same here. I would avoid all changes related to the integration.

File: src/lib/y2users/users_module/reader.rb [hunk 3]
```diff
@@ -33,15 +33,17 @@ module Y2Users
   module UsersModule
     # Class for reading the configuration from Yast::Users module
     class Reader
-      # Generates a new config with the information from YaST::Users module
+      # Generates a new configs with the information from YaST::Users module
       #
-      # @return [Config]
+      # @return [Config,Config] returns pair of configs. system one and modified one
```
[the remark this answers is not in the document]
Comment (jreidinger) [reply src/lib/y2users/users_module/reader.rb:38]: well, I want to emphasize that it is pair and not just random array. Sadly I do not find any example of such yardoc tag.

File: src/include/users/complex.rb (hunk 1 above)
Comment (jreidinger) [reply src/include/users/complex.rb:29]: Well, my goal is to make y2users branch working and @ancorgs then adapt it to whatever style fits. Also it is useful that it reports now issues, which really helps with feedback and testing.

**Pull request merged**